src/SocialMind.TestCmd/Program.cs

```diff
@@ -1,2 +1,30 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using SocialMind.Core.Domain;
+using SocialMind.Core.Domain.DataTransferObjects.Gemini;
+using SocialMind.Core.LLMApiServices;
+
+
+namespace SocialMind.TestCmd;
+
+internal class Program
+{
+    public static async Task Main(string[] args)
+    {
+        const string API_KEY = "AIzaSyAZ-oHoZ6roBFaPC08VLlP95ZCYqOypdtA";
+
+        HttpClient httpClient = new();
+
+        //ILanguageModelService chatService = new MistralApiService(httpClient, API_ENDPOINT, API_KEY);
+        ILanguageModelService chatService = new GeminiApiService(httpClient, GeminiConfig.RequestUri, API_KEY);
+
+        try
+        {
+            ResponseDto? response = await chatService.GetResponseAsync<ResponseDto>("How many hours per day?");
+
+            string responseText = response!.Cadndidates.First().Content.Parts.First().Text;
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine("Error: " + ex.Message);
+        }
```

Comment:
We don't actually need a try catch block here, since we don't want to handle the error. If the the code executed fails, then it will automatically prints the error on the console and stop the program, and that's what we need.